src/components/Summary/SubmissionSummary.js

```diff
@@ -67,25 +54,17 @@ const SubmissionSummary = ({
     error,
   } = useAsync(async () => {
     const submissionUrl = new URL(refreshedSubmission.url);
-
-    let promises = [loadSummaryData(submissionUrl), getPrivacyPolicyInfo(submissionUrl.origin)];
-
-    const [summaryData, privacyInfo] = await Promise.all(promises);
-
-    dispatch({type: 'PRIVACY_POLICY_LOADED', payload: privacyInfo});
-
-    return summaryData;
+    return await loadSummaryData(submissionUrl);
   }, [refreshedSubmission.url]);
 
   if (error) {
     console.error(error);
   }
 
-  const onSubmit = async event => {
-    event.preventDefault();
+  const onSubmit = async ({privacy: privacyPolicyAccepted}) => {
```

Comment:
I don't seem to be able to submit a form anymore :thinking: It seems that here the `onSubmit` function still receives an event?

Reply:
shit yeah I knew that I still had to test end-to-end!